lektor/admin/package.json

```diff
@@ -19,7 +19,7 @@
     "extract-text-webpack-plugin": "^3.0.0",
     "file-loader": "^0.11.2",
     "font-awesome": "^4.3.0",
```

Comment:
I think we should depend on `create-react-class` directly. Currently it's indirectly provided by `react-router`.

Reply:
Fixed, I think I assumed react-codemod would add this in automatically, but it didn't.